### ISSUE TITLE
Please Allow The Cutting Plane of "Section View" To Be Re-oriented

### ISSUE THREAD

**Issue by MadMax389** [bambulab/BambuStudio#594]
Unless I'm missing it, there is no way to change the orientation of the section view cutting plane in the support and color painting tools. The plane always originates from the front of the build plate which makes it less useful for getting under occluded overhangs and the like.

![image](https://user-images.githubusercontent.com/96731892/200659112-05bc8898-ccd4-433a-9922-849b6aba7444.png)

**Comment by Benderrodrigu3z** [comment]
am I missing something? you can if you click drag around the object, if this isnt it im not understanding

**Comment by MadMax389** [comment]
Yes, but it doesn't change the direction of the cutting plane. The cutting plane always originates from the front of the build plate. If I wanted the section to originate from a different angle, I can't find a way to do it. In Prusa Slicer you can change the angle of the section with the "reset direction" option.

![image](https://user-images.githubusercontent.com/96731892/200840793-63cbd9f1-90a0-476b-964f-4138053ef8da.png)
![image](https://user-images.githubusercontent.com/96731892/200840930-a8ea8ed5-a624-4742-a0c4-7af383dd1a47.png)
